src/view/com/util/text/Text.tsx

```diff
@@ -74,49 +75,45 @@ export function Text({
     // @ts-ignore
     if (flattened.fontSize) {
       // @ts-ignore
-      flattened.fontSize = flattened.fontSize * fonts.scaleMultiplier
+      flattened.fontSize = Math.round(
```

Comment:
Rounding here might not be strictly necessary, but it feels bad to have a chance of a value like `14.065`, which is one of the font sizes that we observe in the main feed with some font preferences.